plugins/catalog-backend-module-bitbucket/src/index.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 The Backstage Authors
+ * Copyright 2020 The Backstage Authors
```

Comment:
Is this intended? (Probably, yes)

Reply:
Myeah it's just an artifact of how github treats the move - it's actually mixing up files a bit, since the moves are actually inferred rather than stored in git history. Either way, the year actually isn't all that important in the grand scheme of things :) Thanks for being vigilant!